var/spack/repos/builtin/packages/exago/package.py

```diff
@@ -15,6 +15,7 @@ class Exago(CMakePackage, CudaPackage, ROCmPackage):
     git = "https://gitlab.pnnl.gov/exasgd/frameworks/exago.git"
     maintainers = ["ryandanehy", "CameronRutherford", "pelesh"]
 
+    version("1.5.0", branch="develop", submodules=True)
```

Comment:
Yes - all changes to `exago/package.py` should go in a separate PR - when `1.5.0` is released. Right now I'm attempting to test [via spack CI] outstanding changes in my branch - to identify any potential issues [that need fixing].

Reply:
Well, we can certainly get the right commit has added in a separate PR, but I was hoping my other changes were reasonable and could be applied directly to this PR. 

Happy to table suggestions and add them to a different PR if that would be convenient for testing.

Reply:
To be clear - I expect this PR to dwindle down to only one commit  `xsdk: add version @0.8.0`

And I expect all other commits to go into separate PRs. [using  the commits/diffs from this branch]

The trilinos change is already in a separate PR at https://github.com/spack/spack/pull/33533

And changes related to other pkgs [ginkgo, superlu-dist ] will also be in a separate PR - when the new versions are released.

This PR is my attempt to get some early testing via spack CI [for current/outstanding changes ] - hence marked `don't-merged-yet`

However - I can add in any suggested changes to this branch - so that they get tested - with both spack-CI and xsdk-CI

Reply:
I think adding all the suggested changes to this branch would be great.

If we are able to get a release out the door soon, then I can create my own Spack PR to include those changes, and you should be able to cherry-pick commits from there.